luarules/gadgets/unit_xp_range_bonus.lua

```diff
@@ -14,51 +14,53 @@ if not gadgetHandler:IsSyncedCode() then
     return false
 end
 
-
 local SetUnitWeaponState = Spring.SetUnitWeaponState
 local GetUnitExperience = Spring.GetUnitExperience
 local SetUnitMaxRange = Spring.SetUnitMaxRange
 local GetUnitDefID = Spring.GetUnitDefID
 
 local updateList = {}
+local XPDefs = {}
+
+for unitDefID, unitDef in pairs(UnitDefs) do
+	if unitDef.customParams ~= nil then
```

Comment:
This is not needed, pre-processing of unit defs injects an empty `customParams` table into every unit def.

Reply:
It has little to do with what happens in pre-processing, but yes the table is never nil.

Reply:
How do I remove this line on the Web? As I'm on the train for a while.

Reply:
![image](https://user-images.githubusercontent.com/36827423/224501587-502105f7-96ad-463e-a529-b86294f1549f.png)